lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -84,6 +84,14 @@ class Chef::Knife::VsphereVmClone < Chef::Knife::BaseVsphereCommand
          long: '--cvlan CUST_VLANS',
          description: 'Comma-delimited list of VLAN names for network adapters to join'
 
+  option :customization_sw_uuid,
+         long: '--sw-uuid SWITCH_UUIDS',
+         description: "Distributed virtual switch UUIDs for network adapter to connect, use 'auto' to automatically assign"
```

Comment:
Did you want to mention it's comma separated here, too, like you did in the readme?